javascript/javascript1/week2/student-manager.js

```diff
@@ -0,0 +1,47 @@
+//student manager
+const class07Students = [];
+function addStudentToClass(studentName) { 
+    if(studentName==='Queen') {
+        class07Students.push(studentName);
+    }
+  else if(studentName===''){
+      console.log('Please enter a valid name');
+  }
+  
+  else if(class07Students.length>=6){
```

Comment:
```suggestion
  else if(class07Students.length>7){
```

Reply:
I changed it to - else if(class07Students.length>5) -  as i can only have max of length 6.

Reply:
sorry. i ment 6